### DISCUSSION TITLE
How much time need to transfer in a domain?

### DISCUSSION THREAD

**Discussion by yannglt** [vercel/community#5270] [Feature Question]
### Question

I recently transferred a domain and noticed that, even after a few minutes, it is still marked as registered by a third party, as is the namespace. I understand that transfers like this can take time, but I'm curious to know how long it might typically take.
<img width="1287" alt="image" src="https://github.com/vercel/community/assets/16894359/0b497a15-131d-45d5-becf-1327499184b3">

**Comment by gustavocidornelas** [comment]
hey @yannglt, did it work? If so, how long did it take? I'm having similar issues -- already waited for a few days after initiating the transfer in

**Reply by yannglt** [reply]
Yes it worked. The issue was coming from my previous provider with the confirmation email - had to wait 24h. 
Once confirmed, it took less than 2h for Vercel to make the transfer.

**Reply by z4nr34l** [reply]
Generally, public DNS servers have up to 48h to propagate changes. If you want to make changes in DNS faster please consider using DNS AnyCast on your domain. :D
